minecraft-common/src/main/java/com/sigmundgranaas/forgero/minecraft/common/toolhandler/block/selector/RadiusVeinSelector.java

```diff
@@ -15,7 +15,7 @@
  * <p>
  * Will exclude blocks that are not valid, and try to find valid diagonal blocks.
  *
- * @author StevePlays
+ * @author StevePlays28
```

Comment:
The p in `Steveplays28` shouldn't be capitalised. No big deal, though would appreciate this last small change. The rest looks good though, thanks!

Reply:
Aahaha that's true! I have been writing too much camelcase now it seems xD

Reply:
xD no worries man.